### PR TITLE
Make upload-to-gcs.sh work on repos besides kubernetes/kubernetes

### PR DESCRIPTION
**Pull request by ixdy**
Fix-up after #230.

@spxtr @fejta

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml [hunk 1]
```diff
@@ -214,7 +214,7 @@
         - ensure-upload-to-gcs-script
         - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/_tmp/upload-to-gcs.sh"
         - shell: |
-              if [[ -n '{git-basedir}' ]]; then cd '{git-basedir}'; done
+              if [[ -n '{git-basedir}' ]]; then cd '{git-basedir}'; fi
```
Comment (spxtr) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: whoops

**Comment by spxtr** [comment]
LGTM

**Comment by fejta** [comment]
LGTM

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml (hunk 1 above)
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: *hangs head in shame*

**Pull request merged**